src/main/kotlin/no/nav/familie/ks/sak/config/BisysConfig.kt

```diff
@@ -24,8 +24,13 @@ class BisysConfig(
                 response: HttpServletResponse,
                 filterChain: FilterChain,
             ) {
-                val clientNavn = oidcUtil.getClaim("azp_name")
-                val erKallerBisys = clientNavn.contains("bidrag")
+                val clientNavn: String? =
```

Comment:
Støtte for at token kan mangle azp_name

Reply:
Litt usikker hva `erKallerBisys ` er / var / ble brukt til? Men antar at du har full kontroll..

Reply:
@marius-nav sjekker om token tilhører bisys, men den er avhengig av azp_name eksisterer